scipy/stats/tests/test_distributions.py

```diff
@@ -1175,6 +1175,17 @@ def test_sf(self, x, ref):
     def test_isf(self, q, ref):
         assert_allclose(stats.halflogistic.isf(q), ref, rtol=1e-15)
 
+    @pytest.mark.parametrize("rvs_loc", [1e-5, 1e10])
+    @pytest.mark.parametrize("rvs_scale", [1e-2, 100, 1e8])
+    def test_fit_MLE_comp_optimizer(self, rvs_loc, rvs_scale):
+
+        rng = np.random.default_rng(6762668991392531563)
+        data = stats.halflogistic.rvs(loc=rvs_loc, scale=rvs_scale, size=1000,
+                                      random_state=rng)
+
+        _assert_less_or_close_loglike(stats.halflogistic, data,
+                                      stats.halflogistic.nnlf)
```

Comment:
For simplicity, since it's the default.

```suggestion
        _assert_less_or_close_loglike(stats.halflogistic, data)
```